src/components/LinkCards/LinkCards.module.scss

```
@@ -36,3 +36,16 @@
     line-height: 1.4;
   }
 }
+
+.darkMode {
+  background-color: #2a2a2cdd;
```

Just a small suggestion, as I did not know that hex colours with 8 characters exist. Just learnt something 😄

```suggestion
  // 8-character hex color code that includes alpha channel for transparency
  background-color: #2a2a2cdd;
```